gspread/worksheet.py

```diff
@@ -1371,9 +1371,14 @@ def _finder(self, func, query, in_row=None, in_column=None):
         cells = self._list_cells(values, in_row, in_column)
 
         if isinstance(query, str):
-            match = lambda x: x.value == query
```

Comment:
Do you know why we can't have a lambda here? 🤔Is this from flake8?

Reply:
E731 | do not assign a lambda expression, use a def

https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes

https://www.python.org/dev/peps/pep-0008/#programming-recommendations